config/initializers/wrap_parameters.rb

```diff
@@ -5,7 +5,7 @@
 
 # Enable parameter wrapping for JSON. You can disable this by setting :format to an empty array.
 ActiveSupport.on_load(:action_controller) do
-  wrap_parameters format: [:json]
+  wrap_parameters format: []
```

Comment:
why you change this?

Reply:
We were playing with permitions in params. Also we had strange registration param we were not sending in the frontend, changed it to see what will happen and forgot to put it back))

Reply:
allright! :)